iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/ValidationLevelLogsTest.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.commons.test.TestUtil;
+import com.powsybl.iidm.network.*;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ * @author José Antonio Marqués {@literal <marquesja at aia.es>}
+ */
+
+class ValidationLevelLogsTest {
+
+    @Test
+    void equipmentTest() throws IOException {
+        Network network = networkAssociatedWithReportNode();
+
+        network.setMinimumAcceptableValidationLevel(ValidationLevel.EQUIPMENT);
+        createNetworkWithEquipmentValidationLevel(network);
+
```

Comment:
Maybe assert at this point that no validation messages have been produced. Something like:

```java
// We check that the SILENT option for action on error works
// While creating only network equipment if we have established validation level to EQUIPMENT,
// We should not see anything reported
assertTrue(checkReportNode("""
      Test validationLevel Logs
      """, network.getReportNodeContext().getReportNode()));
```

Reply:
Done